GVFS/GVFS.Common/Maintenance/PostFetchStep.cs

```diff
@@ -26,12 +26,12 @@ protected override void PerformMaintenance()
                 string multiPackIndexLockPath = Path.Combine(this.Context.Enlistment.GitPackRoot, MultiPackIndexLock);
                 this.Context.FileSystem.TryDeleteFile(multiPackIndexLockPath);
 
-                this.RunGitCommand((process) => process.WriteMultiPackIndex(this.Context.Enlistment.GitObjectsRoot));
+                this.RunGitCommand((process) => process.WriteMultiPackIndex(this.Context.Enlistment.GitObjectsRoot), "WriteMultiPackIndex");

```

Comment:
Minor: For these string literals you could switch to `nameof(<method>)` to make future refactoring a bit easier.

Reply:
Done.  I couldn't nameof(GitProcess.WriteMultiPackIndex) out of the box since we used the same name for a local var and the class.  I renamed the local var for ease.